### ISSUE TITLE
Killing civ give more recruits

### ISSUE THREAD

**Issue by zalexki** [Sparker95/Vindicta#330]
It make sens that instability still raise but being able to recruit the neighbors that you just shot doesnt :/ 

I would just not raise (opr maybe decrease) instability when player kill civies.

**Comment by billw2012** [comment]
Yeah I want to make it decrease instability. Thus you are motivated also to help out the civilians you see, not just not kill them.
